projects/distributed-tracing/src/shared/dashboard/dashboard-wrapper/navigable-dashboard.component.test.ts

```diff
@@ -13,16 +19,55 @@ import { ApplicationAwareDashboardComponent } from './application-aware-dashboar
 import { NavigableDashboardComponent } from './navigable-dashboard.component';
 
 describe('Navigable dashboard component', () => {
+  const attributes: FilterAttribute[] = [
```

Comment:
These aren't used by the test are they? If not, let's just provide an empty array. The more we add things like this to test, the larger the impact of a change to `FilterAttribute`

Reply:
This comment still applies I believe

Reply:
Did you refresh? Looks updated to me.